parlai/core/metrics.py

```diff
@@ -103,13 +112,30 @@ def _bleu(guess, answers):
     )
 
 
+def _rouge(guess, answers):
+    """Compute ROUGE score between guess and *any* answers. Return the best."""
+    if rouge is None:
+        return None, None, None
+    evaluator = rouge.Rouge(metrics=['rouge-n', 'rouge-l'], max_n=2)
+    scores = [evaluator.get_scores(normalize_answer(guess), normalize_answer(a))
+              for a in answers]
+    scores_rouge1 = [score['rouge-1']['r'] for score in scores]
```

Comment:
if you choose to make n optional, you'll need to upgrade this code to do the full range of n.